src/main/scala/com/databricks/labs/overwatch/pipeline/BronzeTransforms.scala

```diff
@@ -741,7 +741,7 @@ trait BronzeTransforms extends SparkSessionWrapper {
       val pathsGlob = validNewFilesWMetaDF
         .filter(!'failed && 'withinSpecifiedTimeRange)
         .orderBy('fileSize.desc)
-        .select('fileName)
+        .select('filename)
```

Comment:
I'm not sure how you got this change. It's a good change and one I was planning to implement but curious how you got it. Please leave it for case sensitivity reasons.

Reply:
this needs to be filename all lower case, switched it back to all lower case.